### DISCUSSION TITLE
[Icon Request] Double Check

### DISCUSSION THREAD

**Discussion by ps73** [tailwindlabs/heroicons#466] [Ideas]
A double check icon for indicating read / received status like in whatsapp...
![WhatsAppIcon](https://user-images.githubusercontent.com/17291660/95470339-f7a23a00-0980-11eb-9c8d-7c36d737b30d.png)

**Comment by kishore007k** [comment]
I would like to work on this. If this is still available @ps73 @RobinMalfait @sschoger @adamwathan

**Comment by N0N4M3BNS** [comment]
no update about this request?

**Comment by ndri** [comment]
I added `check-double` to [Sidekickicons](https://github.com/ndri/sidekickicons/), which are meant to be used as a companion to Heroicons: https://sidekickicons.com/?icon=check-double
